integration_tests/seeds/dbt_project_evaluator_exceptions.csv

```diff
@@ -1,2 +1,3 @@
 fct_name,column_name,id_to_exclude,comment
-fct_direct_join_to_source,parent_id,model.dbt_project_evaluator_integration_tests.stg_model_1,This is actually OK because...
\ No newline at end of file
+fct_direct_join_to_source,parent_id,model.dbt_project_evaluator_integration_tests.stg_model_1,This is actually OK because...
+%,%,%exception_model_%,All tests of exception_model are exceptions...
```

Comment:
This test slows down ci. If you don't want to slow it down I will narrow the range.